Description: Fix for ftbfs on mips and mipsel architectures
  This patch adds arch_mips to the list of architectures in order to fix:

  server.c: In function 'handle_probe_cmd':
  server.c:411:15: error: 'arch_mips' undeclared (first use in this function)

  Sven Hoexter suggested the fix and Jens Axboe confirmed it.
Author: Martin Steigerwald <ms@teamix.de>
--- a/arch/arch.h
+++ b/arch/arch.h
@@ -19,6 +19,7 @@
 	arch_arm,
 	arch_sh,
 	arch_hppa,
+	arch_mips,
 
 	arch_generic,
 
